decidim-system/app/views/decidim/system/organizations/edit.html.erb

```diff
@@ -12,6 +12,10 @@
     <p class="help-text"><%= t(".secondary_hosts_hint") %></p>
   </div>
 
+  <div class="field">
+    <%= f.collection_select :available_authorizations, Decidim.authorization_handlers, :name, :name, {}, multiple: true %>
```

Comment:
We should probably use checkboxes instead. Humans don't understand multiple selects.